SU-HSDO/suhumsci
# fix(106):  horizontal card chisel work

# READY FOR REVIEW

## Summary

TODO still trying to figure out URL / external link issue, how to debug

- Categories: make bar the same height as text (maybe use pseudo element with fixed height)    
  - use 600 weight
   - different link underline style
  - little more margin between it and heading
- Image: hover: make it zoom in   
- Review text size across screen breakpoints (lg to sm)

## Steps to Test
1. Need to do design review with Merani or J 👍 
2. Inspect styles, ensure that the news content looks good in the horizontal card view:
- Add a news test page to your local drupal:
- On an existing site (using the colorful theme), create a few pieces of dummy “News Item” content. Remember to add some without images, one without an external link, etc.
- Create an empty “basic page.”
- On the basic page, add a paragraph:
  - View → News
    - Display: → List
- Ensure that the news items are being displayed on the page and the default looks good
(Note that layout is not created yet, so everything will be 100%)
3. inside humsci_basic theme folder, run `npm run test` and make sure everything passes

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
